website/src/services/backendClient.ts

```diff
@@ -2,18 +2,16 @@ import axios, { AxiosError, type Method } from 'axios';
 import { err, ok, Result } from 'neverthrow';
 import { z, ZodSchema } from 'zod';
 
-import { getInstanceLogger, type InstanceLogger } from '../logger.ts';
 import {
     dataUseTermsHistoryEntry,
     getSequencesResponse,
     info,
+    requestUploadResponse,
     sequenceEntryToEdit,
     type ProblemDetail,
 } from '../types/backend.ts';
 import { createAuthorizationHeader } from '../utils/createAuthorizationHeader.ts';
 
-const myLogger = getInstanceLogger('BackendClient');
```

Comment:
What's the reason we're getting rid of the logger?

Reply:
The logger.ts file only works server side and the backend client is now used client side as well. This is related to this PR: https://github.com/loculus-project/loculus/commit/da4e58f79259013cf83bcb1cbc4c556564466536

We got rid of the Zodius stuff, which could use the `logger` field out of the box (if I'm reading this right) but now the field is actually not used.

We could probably make an effort to enhance the logging, but I think it's not urgent.